examples/ct/structures_test.go

```diff
@@ -37,16 +37,16 @@ func TestGetCTLogID(t *testing.T) {
 		t.Fatalf("unexpected error loading public key: %v", err)
 	}
 
-	expected := ctTesttubeLogID
 	got, err := GetCTLogID(km)
 	if err != nil {
 		t.Fatalf("error getting logid: %v", err)
 	}
 
-	got64 := base64.StdEncoding.EncodeToString(got[:])
+	expected := ctTesttubeLogID
+	gotHex := hex.EncodeToString(got[:])
 
-	if expected != got64 {
-		t.Fatalf("expected logID: %s but got: %s", expected, got64)
+	if expected != gotHex {
+		t.Fatalf("expected logID: %s but got: %s", expected, gotHex)
```

Comment:
Fixed this one, but we could probably do with an overall pass to downgrade a bunch more.